packages/database-editor/src/components/ImportWizard/WizardContent.tsx

```diff
@@ -107,13 +30,21 @@ export const WizardContent = ({ context }: { context: DatabaseEditorContext }) =
         >
           {t('import.back')}
         </Button>
-        {activePage !== pages.length - 1 ? (
+        {activePage !== pages.length - 2 ? (
           <Button disabled={!pages[activePage]?.requiredData} variant='primary' size='xl' onClick={() => updateActivePage()}>
-            {t('import.next')}
+            {activePage === pages.length - 1 ? t('import.close') : t('import.next')}
             <IvyIcon icon={IvyIcons.Chevron} />
           </Button>
         ) : (
-          <Button disabled={!pages[activePage]?.requiredData} variant='primary' size='xl' onClick={notImplemented}>
+          <Button
+            disabled={!pages[activePage]?.requiredData}
+            variant='primary'
+            size='xl'
+            onClick={() => {
+              creationFunction.mutate();
+              updateActivePage();
+            }}
+          >
```

Comment:
hard to ready logic...
I think it would be easier to have this inside a new component, with a simple 
if this then return this button
if this then return this button
etc.
instead of this ternary logic...